src/model/fsm_util.jl

```diff
@@ -184,22 +184,65 @@ end
 
 
 """
-Populate fs0m_vec and fs1m_vec for all sources for a given pixel.
+Populate fs0m and fs1m for source s in the a given pixel.
 
-Args:
-    - model_vars: Model intermediate values.
```

Comment:
`tmp_model_vars` might be an event better name for `model_vars`, or `tmp_vars`, just because `model_vars` isn't so clearly different from model parameters, but up to you of course.